homeassistant/components/emoncms/const.py

```diff
@@ -0,0 +1,73 @@
+"""Emoncms constants."""
+
+from dataclasses import dataclass
+
+from homeassistant.components.sensor import SensorDeviceClass, SensorStateClass
+
+
+@dataclass
+class SensorEmoncmsDescription:
+    """Description of an emoncms sensor."""
+
+    device_class: SensorDeviceClass
+    state_class: SensorStateClass
```

Comment:
This is specific for the sensor module, so this shouldn't be here

Reply:
done